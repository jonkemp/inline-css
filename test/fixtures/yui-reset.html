<html>
  <head><link rel="stylesheet" href="yui-reset.css" />
  </head>
  <body>
    <h1>hello</h1></body></html>
