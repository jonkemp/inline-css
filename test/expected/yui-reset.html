<html style="background: #FFF; color: #000;">
  <head>
  </head>
  <body style="margin: 0; padding: 0;">
    <h1 style="font-size: 100%; font-weight: normal; margin: 0; padding: 0;">hello</h1>
  </body>
</html>
